wolfssl/wolfcrypt/asn.h

```diff
@@ -2864,6 +2869,8 @@ struct DecodedCRL {
     word32  sigParamsLength;         /* length of signature parameters   */
 #endif
     byte*   signature;               /* pointer into raw source, not owned */
+    byte    crlNumber[CRL_MAX_NUM_SZ];      /* CRL number extension */
+    byte    crlNumberSet;                   /* CRL number set indicator */
```

Comment:
Assume no customer directly deference this structure or `CRL_Entry`... and since we are changing this around... perhaps you can consolidate the bit fields and use them at the bottom? This would reduce size of these structures.

Like:

```
byte extAuthKeyIdSet:1;
byte crlNumberSet:1;
```

Reply:
Yes. If users are currently directly referencing the `crlNumber`, they'll have to pass that into an `atoi()` call to get it building and restore the same functionality. The pool of users affected by this should be minimal, if not 0. But if we want to get rid of this risk completely, I could keep the `crlNumber` field as an int and add in a `crlNumberLong` field. My preference would be the former